pkg/api/types.go

```diff
@@ -337,6 +354,13 @@ const (
 	// used for PersistentVolumes where the bound PersistentVolumeClaim was deleted
 	// released volumes must be recycled before becoming available again
 	VolumeReleased PersistentVolumePhase = "Released"
+	// used for PersistentVolumes that have been recycled and can be made available
+	// again to new PersistentVolumeClaims
+	VolumeRecycled PersistentVolumePhase = "Recycled"
+	// used for PersistentVolumes that have been removed from the infrastructure
+	VolumeDeleted PersistentVolumePhase = "Deleted"
```

Comment:
If the object is deleted, how does it have a Phase?  Or does this mean that the underlying PD or whatnot is gone, but the kube representation is still here?  Why not just delete it?  Who is supposed to delete it?